journalism_workflow/static/journalism_workflow/copy_editor/partials/copy_editor.html

```diff
@@ -0,0 +1,63 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Finalize the Article Copy
+        </h3>
+      </div>
+    </div>
+    <div class="row padded">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+
+	<!-- Headline -->
+	<div class="row padded">
+	  <div class="col-lg-12 col-md-12 col-sm-12">
+	    <h3>Write a Headline for the Article:</h3>
```

Comment:
pure style: i think imperative commands shouldn't capitalize the whole line, just the first letter of the sentence.  but you can ignore this if you feel the slightest bit in disagreement :)